crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -221,6 +251,10 @@ pub struct LightClientCircuitOutput {
     /// The method id of the light client proof
     /// This is used to compare the previous light client proof method id with the input (current) method id
     pub light_client_proof_method_id: [u32; 8],
+    /// Batch proof info from current or previous light client proofs that were not changed and unable to update the state root yet
+    pub unchained_batch_proofs_info: Vec<BatchProofInfo>,
+    /// Last l2 height the light client proof verifies
+    pub last_l2_height: u64,
```

Comment:
you might think we don't need to output the l2_genesis_root

but we have to, or else we can generate a different state root starting from a different genesis

Reply:
the genesis root output of LCP will be asserted in the bridge itself